### ISSUE TITLE
File Path

### ISSUE THREAD

**Issue by sudip70** [slsdetectorgroup/reuss#36]
There seems to be an issue with file writing as the file path is hard coded rather than getting it from the current directory. 
Since @erikfrojdh  said we are adding a faster drive to keep up with the output. It shouldn't be a big issue or fix.

**Comment by erikfrojdh** [comment]
I think it would be good if the GUI took the current path as the startup value

**Comment by sudip70** [comment]
Yeah, that sounds better. I have added the current directory as the initial file path. I will push the changes into the current PR
